dockerfiles/artifactory/Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM node:8-slim
+
+MAINTAINER Snyk Ltd
+
+# Install broker
+RUN npm install --global snyk-broker
+
+# Don't run as root
+WORKDIR /home/node
+USER node
+
+# Generate default accept filter
+RUN broker init artifactory
+
+
+
+######################################
+# Custom Broker Client configuration #
+# Redefine in derived Dockerfile,    #
+# or provide as runtime args `-e`    #
+######################################
+
+# Your unique Broker identifier
+ENV BROKER_TOKEN <broker-token>
+
+# Your Artifactory URL to artifactory and repository
+# NOTA BENE: without `/` character in the end of URL
+ENV ARTIFACTORY_URL http://yourdomain.artifactory.com/your/repository/name
```

Comment:
Others don't include a path, I imagine that shouldn't be defined here? Also should it be over https?
i.e. change to `https://<your-domain>.artifactory.com`